ray-operator/config/samples/kubectl-ray-submit-job/runtime-env-sample.yaml

```diff
@@ -0,0 +1,6 @@
+pip:
+  - requests==2.26.0
+  - pendulum==2.1.2
+env_vars:
+  counter_name: "test_counter"
+working_dir: ../../../ray-operator/config/samples/kubectl-ray-submit-job/
```

Comment:
I don't think this file, along with the other YAML files, is worth placing in the samples folder. Instead, they should be put in the `testdata` folder alongside the tests. `testdata` is a special folder in Go. See `ray-operator/test/e2e/testdata/` as an example.

Reply:
PTAL, created `/testdata` dir to contain all the YAML files and code needed for ray job submit.